.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,65 @@
+name: Build and test libfsm
+on:
+  pull_request: {}
+  workflow_dispatch:
+    inputs:
+      runner:
+        description: Which runner to use
+        type: choice
+        default: 'ubuntu-22.04'
+        required: true
+        options:
+          - 'ubuntu-22.04'
+          - 'large-ubuntu-22.04-xxl'
+  schedule:
+    - cron: '37 13,1 * * *'
```

Comment:
Do we need a scheduled run?
I don't see it in https://github.com/EVerest/libtimer/pull/8

It would be good to keep this consistent across the various workflows. Note that `on` is not part of the reusable workflow

Reply:
Removed.